internal/cluster/distribution/eks/eksprovider/workflow/activity_calculate_node_pool_version.go

```diff
@@ -62,11 +64,59 @@ func (a CalculateNodePoolVersionActivity) Execute(
 	}
 
 	return CalculateNodePoolVersionActivityOutput{
-		Version: eks.CalculateNodePoolVersion(
+		Version: calculateNodePoolVersion(
 			input.Image,
 			volumeEncryption,
 			fmt.Sprintf("%d", input.VolumeSize),
 			strings.Join(input.CustomSecurityGroups, ","),
 		),
 	}, nil
 }
+
+// CalculateNodePoolVersion retrieves the calculated nodePoolVersion
+//
+// This is a convenience wrapper around the corresponding activity.
+func CalculateNodePoolVersion(
```

Comment:
I think wrappers should be unexported for now, that's how I did the rest - there are a few exceptions for backward compatibility reasons, but in general they should be unexported, the package's interface is much more self-explanatory that way IMO and also we only have a handful of activities/workflows outside this package which calls these activities and those are to be refactored anyway so we don't loose much convenience either.
(I'm still evaluating whether exporting is permissible/preferable, but didn't have much time lately to think about it.)

I know it makes it a bit more difficult to use in the current update workflow, but it's going to be refactored and moved here and then can be simplified using the unexported wrapper.